resources/js/Shared/Layout/Nav.vue

```diff
@@ -46,6 +46,7 @@ function login() {
     onSuccess: () => {
       toggleAuthDialog()
       loginForm.reset()
+      window.location.reload()
```

Comment:
Why are you reloading URL?

Reply:
Component responsible for adding to favorites doesn`t load after login that`s why reloading is needed.
This is probably due to axios, and i am in process of moving to inertia so this should resolve some problems including this one.